Gemfile

```diff
@@ -28,7 +28,7 @@ gem 'sprockets-rails'
 # (see: http://guides.rubyonrails.org/4_2_release_notes.html#respond-with-class-level-respond-to)
 gem 'responders', '~> 2.0'
 
-gem 'sinatra', '~> 2.0.0.beta2', require: 'sinatra/base'
```

Comment:
hah, we've been on a beta for the last three years?

Reply:
yup :upside_down_face: